cmake/Modules/LocalRav1e.cmake

```diff
@@ -99,6 +99,10 @@ else()
 
     set(RAV1E_LIBRARIES ${Rust_CARGO_TARGET_LINK_NATIVE_LIBS})
     if(WIN32)
+        # If an item starts with "/" (e.g., "/defaultlib:msvcrt"), change the
+        # leading "/" to "-" so that the target_link_libraries() call below
+        # will treat the item as a linker flag.
```

Comment:
See https://cmake.org/cmake/help/latest/command/target_link_libraries.html#overview

> * **A link flag**: Item names starting with -, but not -l or -framework, are treated as linker flags. ...

Reply:
I wonder if we should file a cmake bug report about this issue. On Windows, linker flags may start with /. Obviously a concern is that an absolute pathname of a library also starts with / if the drive letter is omitted.

Or we can ask Rust to use `-defaultlib:msvcrt` instead of `/defaultlib:msvcrt`.

Reply:
It sounds like in newer versions of cmake this flag would be better placed in LINK_OPTIONS. It looks like `-nodefaultlibs` is used elsewhere in the rust codebase, so maybe this was just an oversight.

Reply:
Yes, I think we can replace this fix with the following:

```
        set(RAV1E_LINK_OPTIONS ${RAV1E_LIBRARIES})
        list(FILTER RAV1E_LIBRARIES EXCLUDE REGEX "/.*")
        list(FILTER RAV1E_LINK_OPTIONS INCLUDE REGEX "/.*")
        target_link_options(rav1e::rav1e INTERFACE "${RAV1E_LINK_OPTIONS}")
```

Let me merge this fix first, and then experiment with the alternative fix.